bootstrap.yaml

```diff
@@ -0,0 +1,23 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 127.0.0.1, port_value: 9901 }
+
+dynamic_resources:
+  lds_config:
+    api_config_source:
+      api_type: REST
+      cluster_name: [xds_cluster]
+  cds_config:
+    api_config_source:
+      api_type: REST
+      cluster_name: [xds_cluster]
+
+static_resources:
+  clusters:
+  - name: xds_cluster
+    connect_timeout: 0.25s
+    type: STATIC
+    lb_policy: ROUND_ROBIN
+    http_protocol_options: {}
+    hosts: [{ socket_address: { address: 127.0.0.1, port_value: 5000 }}]
```

Comment:
As discussed on slack, we may want to run an xDS per controller and configure this with the controller IPs/ports.